starter_project_backend_nest/src/app.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { MongooseModule } from '@nestjs/mongoose';
+import { AppController } from './app.controller';
+import { AppService } from './app.service';
+import { TodosModule } from './todo/todos.module';
+import { ArticleController } from './article/article.controller';
+import { ConfigModule } from '@nestjs/config';
+
+@Module({
+  imports: [
+    ConfigModule.forRoot(),
+    MongooseModule.forRoot(process.env.MONGO_URI),
```

Comment:
I think it would also be good if we try to extract environment variables in the same way as HakimHub